Libraries/ActionSheetIOS/ActionSheetIOS.js

```diff
@@ -14,8 +14,19 @@
 var RCTActionSheetManager = require('NativeModules').ActionSheetManager;
 
 var invariant = require('invariant');
+var processColor = require('processColor');
 
 var ActionSheetIOS = {
+  processTintColor(tintColor: String) {
```

Comment:
I don't think this check is really needed. processColor does its own input validation, and supports types other than string (such as an array of rgba values).